fts-cron/Dockerfile_cpp

```diff
@@ -23,9 +28,9 @@ RUN yum -y install ca-certificates.noarch lcg-CA voms-clients-cpp fts-rest-cli \
     yum clean all && \
     rm -rf /var/cache/yum
 
-RUN pip install --upgrade pip 
-RUN pip install --upgrade setuptools
-RUN pip install j2cli
+RUN python3 -m pip install --no-cache-dir --upgrade pip
+RUN python3 -m pip install --no-cache-dir --upgrade setuptools
+RUN python3 -m pip install j2cli
```

Comment:
I think you want to add the no-cache-dir here as well (and in the _java file). This just keeps the image size a little smaller.